From 79819c94f27524005889def53740c97a32cd7747 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 16 Apr 2021 20:57:15 +0000
Subject: [PATCH] [unbundle] Use char16_t as UCHAR_TYPE

Overriding UCHAR_TYPE was dropped with:
https://chromium-review.googlesource.com/c/chromium/deps/icu/+/2732628

Bug: 911896
Change-Id: I4c1172aab445c82ba247b1162b8484ed0db9c381
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2830820
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Reviewed-by: Lei Zhang <thestig@chromium.org>
Cr-Commit-Position: refs/heads/master@{#873470}
---
 build/linux/unbundle/icu.gn | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build/linux/unbundle/icu.gn b/build/linux/unbundle/icu.gn
index 0f52fc11e79e6..6f3f8438bd854 100644
--- a/build/linux/unbundle/icu.gn
+++ b/build/linux/unbundle/icu.gn
@@ -16,7 +16,6 @@ config("icu_config") {
   defines = [
     "USING_SYSTEM_ICU=1",
     "ICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_STATIC",
-    "UCHAR_TYPE=uint16_t",
 
     # U_EXPORT (defined in unicode/platform.h) is used to set public visibility
     # on classes through the U_COMMON_API and U_I18N_API macros (among others).
